census/ui_app/EntryForm.jsx

```diff
@@ -141,6 +144,7 @@ const EntryForm = React.createClass({
                   qsSchema={this.yourKnowledgeQSSchema}
                   questions={this.yourKnowledgeQuestions}
                   answers={this.props.answers.aboutYouAnswers}
+                  readonly={readonly}
```

Comment:
Does this take a string here? Wouldn't it be better to have it take a boolean, such as any truthy/falsy value? In JS world, that's quite a common practice :)